<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'>
<head>
<title>mingw-ilmbase.git - mingw-ilmbase</title>
<meta name='generator' content='cgit v0.10.1'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/cgit-data/cgit.css'/>
<link rel='shortcut icon' href='/favicon.ico'/>
<link rel='alternate' title='Atom feed' href='http://pkgs.fedoraproject.org/cgit/mingw-ilmbase.git/atom/ilmbase-2.1.0_obsolete-macros.patch?h=master' type='application/atom+xml'/>
</head>
<body>
<div id='cgit'><table id='header'>
<tr>
<td class='logo' rowspan='2'><a href='/cgit/'><img src='/cgit-data/cgit.png' alt='cgit logo'/></a></td>
<td class='main'><a href='/cgit/'>index</a> : <a title='mingw-ilmbase.git' href='/cgit/mingw-ilmbase.git/'>mingw-ilmbase.git</a></td><td class='form'><form method='get' action=''>
<select name='h' onchange='this.form.submit();'>
<option value='f19'>f19</option>
<option value='f20'>f20</option>
<option value='f21'>f21</option>
<option value='master' selected='selected'>master</option>
</select> <input type='submit' name='' value='switch'/></form></td></tr>
<tr><td class='sub'>mingw-ilmbase</td><td class='sub right'>Jon Ciesla</td></tr></table>
<table class='tabs'><tr><td>
<a href='/cgit/mingw-ilmbase.git/'>summary</a><a href='/cgit/mingw-ilmbase.git/refs/'>refs</a><a href='/cgit/mingw-ilmbase.git/log/ilmbase-2.1.0_obsolete-macros.patch'>log</a><a class='active' href='/cgit/mingw-ilmbase.git/tree/ilmbase-2.1.0_obsolete-macros.patch'>tree</a><a href='/cgit/mingw-ilmbase.git/commit/ilmbase-2.1.0_obsolete-macros.patch'>commit</a><a href='/cgit/mingw-ilmbase.git/diff/ilmbase-2.1.0_obsolete-macros.patch'>diff</a><a href='/cgit/mingw-ilmbase.git/stats/ilmbase-2.1.0_obsolete-macros.patch'>stats</a></td><td class='form'><form class='right' method='get' action='/cgit/mingw-ilmbase.git/log/ilmbase-2.1.0_obsolete-macros.patch'>
<select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='text' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/cgit/mingw-ilmbase.git/tree/'>root</a>/<a href='/cgit/mingw-ilmbase.git/tree/ilmbase-2.1.0_obsolete-macros.patch'>ilmbase-2.1.0_obsolete-macros.patch</a></div><div class='content'>blob: 8140b914ab29d9c020e6f9a69edd026daaa191c1 (<a href='/cgit/mingw-ilmbase.git/plain/ilmbase-2.1.0_obsolete-macros.patch'>plain</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a id='n1' href='#n1'>1</a>
<a id='n2' href='#n2'>2</a>
<a id='n3' href='#n3'>3</a>
<a id='n4' href='#n4'>4</a>
<a id='n5' href='#n5'>5</a>
<a id='n6' href='#n6'>6</a>
<a id='n7' href='#n7'>7</a>
<a id='n8' href='#n8'>8</a>
<a id='n9' href='#n9'>9</a>
<a id='n10' href='#n10'>10</a>
<a id='n11' href='#n11'>11</a>
<a id='n12' href='#n12'>12</a>
<a id='n13' href='#n13'>13</a>
<a id='n14' href='#n14'>14</a>
<a id='n15' href='#n15'>15</a>
<a id='n16' href='#n16'>16</a>
<a id='n17' href='#n17'>17</a>
<a id='n18' href='#n18'>18</a>
<a id='n19' href='#n19'>19</a>
<a id='n20' href='#n20'>20</a>
<a id='n21' href='#n21'>21</a>
</pre></td>
<td class='lines'><pre><code>diff -rupN ilmbase-2.1.0/configure.ac ilmbase-2.1.0-new/configure.ac
--- ilmbase-2.1.0/configure.ac	2013-11-12 00:09:51.000000000 +0100
+++ ilmbase-2.1.0-new/configure.ac	2014-01-13 16:32:49.754639829 +0100
@@ -10,7 +10,7 @@ AC_SUBST(ILMBASE_VERSION_API, ${ILMBASE_
 
 AC_CANONICAL_HOST
 AC_CONFIG_SRCDIR(ImathTest/main.cpp)
-AM_CONFIG_HEADER(config/IlmBaseConfig.h)
+AC_CONFIG_HEADERS(config/IlmBaseConfig.h)
 AM_INIT_AUTOMAKE(1.6.3)  dnl Require automake 1.6.3 or better
 AM_MAINTAINER_MODE
 
@@ -26,7 +26,7 @@ AC_PROG_CXX
 AC_PROG_INSTALL
 AC_PROG_CC
 AC_PROG_LN_S
-AC_PROG_LIBTOOL
+LT_INIT
 AC_PROG_MAKE_SET
 
 dnl
</code></pre></td></tr></table>
</div> <!-- class=content -->
<div class='footer'>generated  by cgit v0.10.1 at 2014-08-28 07:12:12 (GMT)</div>
</div> <!-- id=cgit -->
</body>
</html>
